On macOS I get two test failures when running `test_os`:
```
======================================================================
FAIL: test_pathlike_class_getitem (test.test_os.TestPEP519)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/guido/pep585/Lib/test/test_os.py", line 4083, in test_pathlike_class_getitem
    self.assertIs(os.PathLike[bytes], os.PathLike)
AssertionError: os.PathLike[bytes] is not <class 'os.PathLike'>

======================================================================
FAIL: test_pathlike_class_getitem (test.test_os.TestPEP519PurePython)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/guido/pep585/Lib/test/test_os.py", line 4083, in test_pathlike_class_getitem
    self.assertIs(os.PathLike[bytes], os.PathLike)
AssertionError: os.PathLike[bytes] is not <class 'os.PathLike'>
```
(I realize that one weakness of my setup for this branch is that CI doesn't run for PRs.)

--- a/Lib/test/test_os.py
+++ b/Lib/test/test_os.py
@@ -25,6 +25,7 @@ import sysconfig
 import tempfile
 import threading
 import time
+import types
 import unittest
 import uuid
 import warnings
@@ -4080,7 +4081,7 @@ class TestPEP519(unittest.TestCase):
         self.assertTrue(issubclass(FakePath, os.PathLike))
 
     def test_pathlike_class_getitem(self):
-        self.assertIsInstance(os.PathLike[bytes], os.PathLike)
+        self.assertIsInstance(os.PathLike[bytes], types.GenericAlias)
 
 
 class TimesTests(unittest.TestCase):

--- a/Lib/test/test_os.py
+++ b/Lib/test/test_os.py
@@ -4080,7 +4080,7 @@ class TestPEP519(unittest.TestCase):
         self.assertTrue(issubclass(FakePath, os.PathLike))
 
     def test_pathlike_class_getitem(self):
-        self.assertIs(os.PathLike[bytes], os.PathLike)
+        self.assertIsInstance(os.PathLike[bytes], os.PathLike)
 
 
 class TimesTests(unittest.TestCase):

